src/components/WalletCard/WalletCard.js

```diff
@@ -25,15 +26,21 @@ type Props = {|
 
 type StateProps = {|
   +newName: string,
+  +isNewNameUniq: boolean,
   +isRenameActive: boolean,
 |}
 
+function sanitizeName(name: string) {
+  return name.substring(0, 32).trim().replace(/\//g, '–')
```

Comment:
May be, replace then trim will be better?

Reply:
Do we need to move this fn to utils to share it?

Reply:
> May be, replace then trim will be better?

it doesn't matter, because there are no any spaces after `replace`

Reply:
> Do we need to move this fn to utils to share it?

these actions are actual during renaming of a wallet only

Reply:
fixed in 374e4df